tools/dispatch_data.py

```diff
@@ -33,23 +33,23 @@ def get_launch_cmd(args) -> str:
 def submit_jobs(args) -> str:
     wrapper_command = os.path.join(INSTALL_DIR, LAUNCH_SCRIPT)
 
-    #read the json file and get the remaining argument here. 
-    schema_path = os.path.join(args.in_dir, "metadata.json")
-    with open(schema_path) as schema:
+    #read the json file and get the remaining argument here.
+    schema_path = "metadata.json"
+    with open(os.path.join(args.in_dir, schema_path)) as schema:
         schema_map = json.load(schema)
 
     num_parts = len(schema_map["num_nodes_per_chunk"][0])
     graph_name = schema_map["graph_name"]
 
     argslist = ""
     argslist += "--world-size {} ".format(num_parts)
-    argslist += "--partitions-dir {} ".format(args.partitions_dir)
-    argslist += "--input-dir {} ".format(args.in_dir)
+    argslist += "--partitions-dir {} ".format(os.path.abspath(args.partitions_dir))
```

Comment:
The doc said we "support absolute path or path relative to the location of metadata.json." Could you please confirm? If not, we may want to reword the doc.

Reply:
I checked. Only absolute paths work. I'll update the doc.

Reply:
Fixed in #4433